### PR TITLE
Add small glossary

### LINKED ISSUE #246: Add small glossary
As @shabanim suggested in the meeting, I think having a small glossary is helpful for new contributors to understand common used words in the context of this repo. I will add a small glossary for some common used words.

@shabanim Feel free to add any words you think we should explain.

Some words in my minds:
* accelerator
* throughput
* latency
* accuracy
* backend
* vendor
* loadgen
* task
* performance mode
* accuracy mode
* single stream scenario
* offline scenario
* benchmark

### PR DESCRIPTION
**Pull request by anhappdev**
Closes #246

### CODE REVIEW

File: docs/glossary.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+| single stream scenario   | The single-stream scenario represents one inference-query stream with a query sample size of 1, reflecting the many client applications where responsiveness is critical. An example is offline voice transcription on Google’s Pixel 4 smartphone. To measure performance, we inject a single query into the inference system; when the query is complete, we record the completion time and inject the next query. The metric is the query stream’s 90th-percentile latency.<br>Source: <https://arxiv.org/pdf/1911.02549.pdf>                                                                                                                                       |
```
Comment (d-uzlov) [inline docs/glossary.md:20]: >The metric is the query stream’s 90th-percentile latency.

I think we should note that in our application the latency is converted to equivalent throughput.

File: docs/glossary.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+| offline scenario         | The offline scenario represents batch-processing applications where all data is immediately available and latency is unconstrained. An example is identifying the people and locations in a photo album. For this scenario, we send a single query that includes all sample-data IDs to be processed, and the system is free to process the input data in any order. Similar to the multistream scenario, neighboring samples in the query are contiguous in memory. The metric for the offline scenario is throughput measured in samples per second.<br>Source: <https://arxiv.org/pdf/1911.02549.pdf>                                                               |
+
+**Helpful links**:
+
+* <https://mlcommons.org/en/inference-mobile-11>
+* <https://github.com/mlcommons/inference_policies/blob/master/inference_rules.adoc>
```
Comment (d-uzlov) [inline docs/glossary.md:26]: Could you add a short description for these links, please?

**Review by d-uzlov — CHANGES_REQUESTED** [review]

File: docs/glossary.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+| LoadGen (Load Generator) | The LoadGen is a traffic generator for MLPerf Inference that loads the SUT and measures performance. Its behavior is controlled by a configuration file it reads at the start of the run. The LoadGen produces the query traffic according to the rules of the previously described scenarios (i.e., single-stream, multistream, server, and offline). Additionally, it collects information for logging, debugging, and postprocessing the data. It records queries and responses from the SUT, and at the end of the run, it reports statistics, summarizes the results, and determines whether the run was valid.<br>Source: <https://arxiv.org/pdf/1911.02549.pdf> |
```
[the remark this answers is not in the document]
Comment (d-uzlov) [reply docs/glossary.md:19]: It would also help to add a link to the loadgen repository.

File: docs/glossary.md (hunk 2 above)
Comment (anhappdev) [reply docs/glossary.md]: Fixed.

File: docs/glossary.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (anhappdev) [reply docs/glossary.md]: Fixed.

File: docs/glossary.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+| backend                  | Vendor-specific code for running benchmark.<br>We have a file naming convention for vendor's backend used in release build:<br>Q - QTI<br>S - Samsung SLSI<br>M - MediaTek<br>G - Google Pixel<br>I - Intel<br>H - Huawei<br>T - TFLite                                                                                                                                                                                                                                                                                                                                                                                                                                |
```
[the remark this answers is not in the document]
Comment (anhappdev) [reply docs/glossary.md]: Fixed. It's now in `build_and_release.md`

File: docs/glossary.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+| accuracy                 | The accuracy of a benchmark. The unit of measurements is depended on benchmark type.<br/>For example, mAP is used for object detection task and F1 score is used for language processing task.                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
```
[the remark this answers is not in the document]
Comment (anhappdev) [reply docs/glossary.md]: Fixed.

**Review by d-uzlov — APPROVED** [review]

**Pull request merged**